README.md

```diff
@@ -191,8 +191,63 @@ Packages found only in gcr.io/google-appengine/python:2017-06-29-190410: None
 Version differences: None
 
 ```
+## Example Run with json post-processing
```

Comment:
I feel like it may be more interesting to include the actual json (I would make a very small sample one, not necessarily based on an existing image or anything) and the results of processing that json than to include the python script itself.  Then you could just say-- I had this json created by running the node, pip, and apt differ on two images, and my python script did x, y, z to produce this output, etc....